docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md

```diff
@@ -7,29 +7,44 @@ title: Rewards Claiming Overview
 
 ## Overview
 
-The EigenLayer Rewards protocol enables AVSs to make rewards to stakers and operators. Operators earn rewards by opting in to AVSs that make `RewardsSubmissions` to the `RewardsCoordinator`, a core protocol contract. Within a single `RewardsSubmission`, an AVS can specify a time range for which the reward will be distributed, a list of weights for each `Strategy` for the reward, and an ERC20 token to make rewards in.  
+The EigenLayer Rewards protocol enables AVSs to make rewards to stakers and operators. Operators earn rewards by opting in to AVSs that make `RewardsSubmissions` to the `RewardsCoordinator`, a core protocol contract. Within a single `RewardsSubmission`,  an AVS can specify a time range for which the reward will be distributed, a list of weights for each `Strategy` for the reward, and an ERC20 token to make rewards in.
 
-Operators will earn a flat 10% commission on rewards. The rest of the reward is passed on to the operator's delegated stakers. Rewards are proportional to:
+By default, Operators will earn a flat 10% split on rewards. The rest of the reward is claimable bythe operator's delegated Stakers. Rewards are proportional to:
```

Comment:
by the ---- need a space

Reply:
Can you clarify what you mean?

Reply:
there is a typo
between the words by and the we need a space on this line

Reply:
good call, resolved now